src/back/controllers/api/routes/profile/myProfile.controller.spec.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const MyProfileController = require('./myProfile.controller');
+const ProfileController = require('./profile.controller');
+const profileProperties = require('./profileProperties');
+
+describe('MyProfileController', () => {
+    let sut;
+
+    beforeEach(() => {
+        sut = new MyProfileController();
+    });
+
+    it('should extend profile controller', () => {
+        sut.should.be.instanceOf(ProfileController);
```

Comment:
I've got your intention, but:

1. All methods of `ProfileController` might be overridden in the constructor or prototypes chain might look like `MyProfileController <- ControllerThatOverridesProfileMethods <- ProfileController`. So this test doesn't guarantee that functionality is the same. 

2. Do you really need **all** methods of `ProfileController` in this controller?